.github/workflows/prt_labels.yml

```diff
@@ -0,0 +1,44 @@
+name: Remove the PRT label, for the new commit
+
+on:
+  pull_request:
+    types: ["opened", "synchronize"]
+
+jobs:
+    prt_labels:
+      name: remove the PRT label, if it is a new commit
+      runs-on: ubuntu-latest
+      steps:
+        - name: Fetch the PRT status
+          id: prt
+          uses: omkarkhatavkar/wait-for-status-checks@main
+          with:
+            ref: ${{ github.head_ref }}
+            context: 'Robottelo-Runner'
+            wait-interval: 2
+            count: 5
+
+        - name: remove the PRT Passed/Failed label, for new commit
+          if: always() && ${{steps.prt.outputs.result}} == 'not_found'
+          uses: actions/github-script@v7
+          with:
```

Comment:
With this change, will it be possible to delete the previous PRT result comment as well, when new commit appears?
In case of rebase, I think we shouldn't delete, but for new commit we could delete it to avoid confusion for results or do you consider to handle it differently, wdyt @omkarkhatavkar (cc @jyejare @JacobCallahan) ?

Reply:
Consider the case where you have two different PRT comments with different tests (collection) and will have different test results. `rebase` would not delete the labels.

Reply:
@Gauravtalreja1 Good Question/Ask. As @omkarkhatavkar stated, deleting the previous PRT result wouldn't be required in all the cases.

But as part of this request, we could do multiple things:
- Add a comment on PR that the PR is refreshed, need to rerun the PRT.
- OR If no PRT specific label on PR, block the PR merge.
- etc.

Reply:
@Gauravtalreja1 if you are okay with change can you please resolve this discussion ?

Reply:
@omkarkhatavkar Yes, I'm okay with the current changes, but I feel it can be surely considered for future, so I think we can have a issue created, or we can keep this thread

And about the first suggestion of @jyejare I think its not good idea as adding such comments will unnecessarily increase the comment history, and for second suggestion, I feel we shouldn't block the PR merge on PRT results ever, as some PRs might not require results like easy fix PRs, syntax fixing PRs, or some dependabot PRs which gets automerged when basic GHA checks pass

Reply:
@Gauravtalreja1 As discussed in yesterday discussion @JacobCallahan also agreed with majority of people having no concern about having multiple PRT result comments. As well as github also have feature of grouping similar comments. I personally dont feel we have to track this as there no action item right now.

Reply:
@Gauravtalreja1 auto merge is not enabled on the master, we never merge the PR on the master. Yesterday I talked about the cherry-picks PR. Adding the automerge will help in merging the PR's quickly. I dont think we would like to merge blocking based on the PRT Failure Result. Current label structure provide more detail to help people as author and reviewer

Reply:
@Gauravtalreja1 @omkarkhatavkar I was just throwing ideas if we really want to strict the PRT results / labels and maintain accurate history. But the real thing is we dont want to delete the RPT history for now.